src/wallet/components/WalletController.tsx

```diff
@@ -74,39 +80,94 @@ export default class WalletController extends PureComponent<Props> {
         return <div>Waiting for challenge</div>;
       case WaitForChallengeConcludeOrExpire:
         return <div>Waiting for opponent to respond to challenge</div>;
+        case playerA.ReadyToDeploy:
+        return <FundingInProgress 
```

Comment:
@lalexgap The changes in this file are a bit hard to read, I wonder if indenting code inside case blocks would help.